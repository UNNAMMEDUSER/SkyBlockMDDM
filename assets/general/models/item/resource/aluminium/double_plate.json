{
	"parent": "item/generated",
	"textures": {
		"layer0": "general:item/resource/aluminium/double_plate"
	}
}