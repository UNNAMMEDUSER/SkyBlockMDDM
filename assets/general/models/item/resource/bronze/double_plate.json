{
	"parent": "item/generated",
	"textures": {
		"layer0": "general:item/resource/bronze/double_plate"
	}
}