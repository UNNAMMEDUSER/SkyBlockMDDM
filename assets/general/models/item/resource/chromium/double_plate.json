{
	"parent": "item/generated",
	"textures": {
		"layer0": "general:item/resource/chromium/double_plate"
	}
}