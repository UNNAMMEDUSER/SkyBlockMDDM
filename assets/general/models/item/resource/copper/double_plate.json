{
	"parent": "item/generated",
	"textures": {
		"layer0": "general:item/resource/copper/double_plate"
	}
}