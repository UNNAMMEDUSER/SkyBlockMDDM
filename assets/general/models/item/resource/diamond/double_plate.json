{
	"parent": "item/generated",
	"textures": {
		"layer0": "general:item/resource/diamond/double_plate"
	}
}