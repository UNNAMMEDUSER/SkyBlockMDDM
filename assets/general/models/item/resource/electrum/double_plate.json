{
	"parent": "item/generated",
	"textures": {
		"layer0": "general:item/resource/electrum/double_plate"
	}
}