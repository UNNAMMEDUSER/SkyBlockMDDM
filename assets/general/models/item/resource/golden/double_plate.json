{
	"parent": "item/generated",
	"textures": {
		"layer0": "general:item/resource/golden/double_plate"
	}
}